src/EditorFeatures/Core/Extensibility/BraceMatching/AbstractDirectiveTriviaBraceMatcher.cs

```diff
@@ -66,10 +66,10 @@ internal abstract class AbstractDirectiveTriviaBraceMatcher<TDirectiveTriviaSynt
 
         private static bool IsConditionalDirective(TDirectiveTriviaSyntax directive)
         {
-            return directive is TIfDirectiveTriviaSyntax ||
-                   directive is TElseIfDirectiveTriviaSyntax ||
-                   directive is TElseDirectiveTriviaSyntax ||
-                   directive is TEndIfDirectiveTriviaSyntax;
+            return directive is TIfDirectiveTriviaSyntax or
+                   TElseIfDirectiveTriviaSyntax or
+                   TElseDirectiveTriviaSyntax or
+                   TEndIfDirectiveTriviaSyntax;
```

Comment:
debatable about what alignment we want.  with the prior style, the checks always line up.  now, the 2-nth check aligns with the expr, not the test case.